### DISCUSSION TITLE
Milvus search performance

### DISCUSSION THREAD

**Discussion by phosphorylation** [milvus-io/milvus#7010] [Q&A and General discussion]
Hi, I submitted a question a week ago regarding the expected performance of Milvus' IVFFlat index. #6866
As I was digging the paper _Milvus: A Purpose-Built Vector Data Management System_  you guys publicated on SIGMOD21, I found this figure which showcases IVFFlat performance. There's a big gap between my results and the paper's, I'd be grateful if someone could reason about it.
![paper figure 8](https://user-images.githubusercontent.com/38056156/128171655-2bbbb1b0-be28-40a7-b266-f8d23913d263.png)

**Comment by yhmo** [comment]
The experiment of the paper is base on milvus v1.x. There is s big performance gap between v2.0 and v1.x, now.

**Reply by hadim** [reply]
What do you mean by performance gap? Is 2.0 faster or slower than 1.x? Even when the vectors are loaded in memory?

I am considering switching to Milvus 2.0 soon but will likely postpone if performance are worse than 1.x.

**Reply by hadim** [reply]
Looks like I got my answer from https://github.com/milvus-io/milvus/discussions/6866

**Comment by xiaofan-luan** [comment]
> Hi, I submitted a question a week ago regarding the expected performance of Milvus' IVFFlat index. #6866
> As I was digging the paper _Milvus: A Purpose-Built Vector Data Management System_ you guys publicated on SIGMOD21, I found this figure which showcases IVFFlat performance. There's a big gap between my results and the paper's, I'd be grateful if someone could reason about it.
> ![paper figure 8](https://user-images.githubusercontent.com/38056156/128171655-2bbbb1b0-be28-40a7-b266-f8d23913d263.png)

Interesting? can you share us the performance benchmark number? 
Also the performance is related to your hardware configuration, any benchmark report you could share to the community?
